modules/rio/hm.nix

```
@@ -15,7 +15,9 @@ in
     programs.rio.settings = {
       fonts = with config.stylix.fonts; {
         family = monospace.name;
-        size = sizes.terminal;
+        emoji.family = emoji.name;
+        # converting font size to px
+        size = sizes.terminal * 4.0 / 3.0 + 0.5;
```

```suggestion
        size = sizes.terminal * 4.0 / 3.0;
```

`+ 0.5` is only necessary if you are rounding with `builtins.floor`. it is not part of the actual conversion from pt to px